### PR TITLE
Added GXX_ROOT to variables in module file as this impacts people who…

### PR DESCRIPTION
**Pull request by ocaisa**
… use locales and intel compilers

### CODE REVIEW

File: easybuild/easyblocks/generic/intelbase.py [hunk 1]
```diff
@@ -358,6 +358,12 @@ def make_module_extra(self):
             nlspath = os.path.join('idb', 'intel64', 'locale', '%l_%t', '%N')
         txt += self.module_generator.prepend_paths('NLSPATH', nlspath)
 
+        # In case people are using unusual locales need to set GXXROOT, see
+        # https://software.intel.com/en-us/articles/
+        # intel-fortran-compiler-for-linux-ifort-error-could-not-find-directory-in-which-g-resides
+        get_gxxroot = "g++ --print-search-dirs | head -n1 | awk '{print $2}'"
```
Comment (ocaisa) [inline easybuild/easyblocks/generic/intelbase.py]: Someone with better fu could probably improve this

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
cleaned up (and fixed) in https://github.com/ocaisa/easybuild-easyblocks/pull/7

**Comment by boegel** [comment]
Going in, thanks @ocaisa!

**Pull request merged**